src/dev_usb.h

```diff
@@ -24,17 +24,17 @@ int open_dev_usb(struct device *dev);
 
 /* USB device enumeration and matching */
 #define MAX_USB_DEV_FILES	16
-struct usb_device_info {
+struct sn_usb_device_info {
```

Comment:
Thank you for pointing out the conflict, but I don't like the name. I'd rather shorten it than make it more verbose. `usb_dev_info` is preferable.

Reply:
Yeah, I hated it too.